packages/orchestration/src/examples/send-anywhere.contract.js

```diff
@@ -66,18 +68,19 @@ export const contract = async (
   const sharedLocalAccountP = zone.makeOnce('localAccount', () =>
     makeLocalAccount(),
   );
+  // same as above.
```

Comment:
Maybe make the reason for the cast to any more specific:
```suggestion
  // UNTIL https://github.com/Agoric/agoric-sdk/issues/9822
```

Reply:
done